examples/enclave_manager/tcf_enclave_manager/enclave_manager.py

```diff
@@ -490,11 +523,23 @@ def start_enclave_manager(config):
 
 
 TCFHOME = os.environ.get("TCF_HOME", "../../../../")
+logger.info(TCFHOME)
+logger.info("This is the TCFHOME in enclave manager")
 
 # -----------------------------------------------------------------
 # -----------------------------------------------------------------
 
 
+def connect_zmq(config):
+    context = zmq.Context()
+    socket = context.socket(zmq.REP)
+    ZMQ_PORT = config.get("Listener")["zmq_port"]
+    logger.info("This is the ZMQ_PORT")
+    logger.info(ZMQ_PORT)
+    socket.bind("tcp://*:" + ZMQ_PORT)
```

Comment:
Will this work if enclave manager and Listener are running in different machines.